server/validation.js

```diff
@@ -254,6 +257,26 @@ const ParticipantBatchSchema = yup.array().of(
   }),
 );
 
+const ParticipantSchema = yup.object().shape({
+  // Eligibility
+  eligibility: yup.boolean().typeError(errorMessage).required(errorMessage).test('is-true', errorMessage, (v) => v === true),
+
+  // Contact info
+  firstName: yup.string().required(errorMessage),
+  lastName: yup.string().required(errorMessage),
+  phoneNumber: yup.mixed().required(errorMessage),
```

Comment:
Can we make sure we get this format validated? I think this is HCAP-347 sneaking in